### PR TITLE
Added parseRawTransaction function

### PR DESCRIPTION
**Pull request by nathanieliov**

### CODE REVIEW

**Review by marcos-iov — CHANGES_REQUESTED** [review]
No tests?

**Comment by nathanieliov** [comment]
> No tests?

@marcos-iov 

The thing is would be worth adding a test which most of it is a mock? Because the test must be reproducible. No bitcoin node to test against it. Need to create a mock mimicking it.

File: btc-transaction-helper.js [hunk 1]
```diff
@@ -272,6 +272,15 @@ class BtcTransactionHelper {
         return Buffer.concat([bufferVersion, decodedAddress.hash]).toString('hex');
     };
 
+    /**
+     *
+     * @param {string} rawTransaction string in hex to be parsed into an object
+     * @param {boolean} isWitness string in hex to be parsed into an object
```
Comment (marcos-iov) [inline btc-transaction-helper.js]: ?

File: test/btc-transaction.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
```
Comment (marcos-iov) [inline test/btc-transaction.json]: Just an idea, we could have all this in a single file called `sample-btc-transactions` or something like that
```
[
 { 
   rawtx: "....",
   decoded: "...."
 },
{ 
   rawtx: "....",
   decoded: "...."
 }
]
```
What do you think?

File: btc-transaction-helper.js (hunk 1 above)
Comment (nathanieliov) [reply btc-transaction-helper.js]: `isWitness` is an optional parameter that can be passed for decoding a raw transaction to the btc node client, but then I realized is not needed. See more info on how it works here: https://chainquery.com/bitcoin-cli/decoderawtransaction

File: test/btc-transaction.json (hunk 2 above)
Comment (nathanieliov) [reply test/btc-transaction.json]: I like the idea, most because it'll reduce all into a single file.

File: btc-transaction-helper.js (hunk 1 above)
Comment (nathanieliov) [reply btc-transaction-helper.js]: Unknown parameter description removed

File: test/btc-transaction.json (hunk 2 above)
Comment (nathanieliov) [reply test/btc-transaction.json]: Done

**Review by marcos-iov — COMMENTED** [review]
Looking good, just one more thing. We need to bump the version number

**Review by marcos-iov — APPROVED** [review]

**Review by kelvinator07 — APPROVED** [review]

**Pull request merged**